_posts/2022-11-21-introducing-intern-robert-min.md

```diff
@@ -1,12 +1,15 @@
 ---
 layout: post
-title: "Introducing Namecoin's New Intern, Robert Min"
-author: "Jeremy Rand and Robert Min"
+title: "Introducing Namecoin's New Intern, Robert Nganga"
+author: "Jeremy Rand and Robert Nganga"
 tags: [News]
 ---
 
-Hey everyone, Jeremy here.  We're happy to announce that Namecoin has accepted an intern via [Outreachy](https://www.outreachy.org/) (we are participating under the Tor Project umbrella).  Robert Min will be funded to work on a new project that I think both the Namecoin and Tor communities will find exciting.  Huge thanks are due to Tor Project for taking us under their wing for this internship.  I don't want to steal his thunder any more than I already have, so I'm going to turn over the remainder of this post to Robert to tell you about his project -- please join us in welcoming him to the Namecoin community!
+Hey everyone, Jeremy here.  We're happy to announce that Namecoin has accepted an intern via [Outreachy](https://www.outreachy.org/) (we are participating under the Tor Project umbrella).  Robert Nganga will be funded to work on a new project that I think both the Namecoin and Tor communities will find exciting.  Huge thanks are due to Tor Project for taking us under their wing for this internship.  I don't want to steal his thunder any more than I already have, so I'm going to turn over the remainder of this post to Robert to tell you about his project -- please join us in welcoming him to the Namecoin community!
 
-[TODO]
+Greetings, I am pleased to join the Namecoin team as a [Outreachy](https://www.outreachy.org/) Intern after two months of contributing to Namecoin. I'll be working on developing a tool that can identify and prevent proxy leaks. I'll be using ptrace to intercept network socket system calls in a Tor-using application and block them if they constitute a proxy leak. Given the substantial threats posed by proxy leaks, the technology will improve security and privacy while resolving current problems with the technologies already in use. For instance, torsocks, an LD PRELOAD-based program that intercepts calls to the network functions of the standard C library, is incompatible with statically linked software, such as the majority of Golang-written applications (Given that Golang is frequently used for security-conscious applications, this is a particularly problematic) Transparent proxy tools like Whonix can result in usage of multiple applications being traceable to the same user, since you can't easily enforce stream isolation (degrading anonymity into much weaker pseudonymity)
```

Comment:
Nits: LD_PRELOAD should have an underscore rather than a space; "problematic" seems to be missing a noun after it; also there should be a blank line after this paragraph so that Jekyll handles formatting properly.

None of these are blockers, I can clean up any formatting        issues such as the blank line once it's merged -- but feel free to fix them if you have time today.

Reply:
Great, twerked it.